### ISSUE TITLE
Auto-modify matching opening/closing tag in HTML

### ISSUE THREAD

**Issue by ussuri** [ajaxorg/ace#2263]
When the cursor moves inside a tag in an HTML source, Ace detects and highlights the matching opening or closing one, if it exists. It would be useful if it could also synchronize editing of the tag pair: if either one is changed, the other changes accordingly. Would it be feasible to implement?

**Comment by matthew-dean** [comment]
Yes, Chrome developer tools will change the ending tag if you modify the opening tag in the "elements" view.

**Comment by levithomason** [comment]
Webstorm also does this out of the box.

**Comment by baktiaditya** [comment]
👍  must-have feature

**Comment by levithomason** [comment]
https://codepen.io released this feature recently as well.

**Comment by ussuri** [comment]
If Ace wants to stay competitive with other IDEs mentioned in the previous comments, it might want to implement this, after all.
